frontend/controller/actions/group.js

```diff
@@ -122,10 +122,12 @@ export default sbp('sbp/selectors/register', {
   },
   'gi.actions/group/updateSettings': async function (settings, groupId) {
     try {
+      this.dumbError.forced = 123 // DELETE THIS BEFORE MERGE
       const message = await sbp('gi.contracts/group/updateSettings/create', settings, groupId)
       await sbp('backend/publishLogEntry', message)
     } catch (e) {
-      throw new GIErrorUIRuntimeError(L('Failed to update group settings. {codeError}', { codeError: e.message }))
+      console.error('gi.actions/group/updateSettings failed!', e)
+      throw new GIErrorUIRuntimeError(L('Failed to update group settings. {reportError}', LError(e)))
```

Comment:
Should read:

```js
this.$refs.formMsg.danger(L('Error updating group settings: {error}. {reportError}', LError(e)))
```

Similar changes throughout rest of file.

Reply:
What if error, comes from {reportError}? That way we don’t need to repeat the 2 variables everywhere.

Reply:
@sandrina-p that's a good idea, ok, in that case this would read:

```js
this.$refs.formMsg.danger(L('Error updating group settings. {reportError}', LError(e)))
```

And then in `LError` you'd have:

```js
{errorMsg}. {a_}See error logs.{_a}
```

Reply:
Cool! Regarding the error itself. I find the wording ‘Error updating group settings’ kinda 🤖. Don’t you think ‘Failed to update group settings’ more human 👀?

Reply:
sure 👍